package/MDAnalysis/__init__.py

```diff
@@ -140,6 +140,7 @@
 import warnings
 
 from .version import __version__
+from .authors import __authors__
```

Comment:
So if setup is ran from a location where you don't have write permission (is this possible?), this file won't be generated.  So would it be safer to wrap this in a try/except block just so this never stops the entire module working one day?